tox.ini

```diff
@@ -17,12 +22,12 @@ whitelist_externals =
 passenv = DATABASE_URL
 skip_install = true
 commands =
-    bash -c 'poetry export --dev --without-hashes -f requirements.txt | grep -v "^[dD]jango==" | pip install -q --no-deps -r /dev/stdin'
+    bash -c 'poetry export --only dev --without-hashes -f requirements.txt | grep -v "^[dD]jango==" | pip install -q --no-deps -r /dev/stdin'
     pip install -q --no-deps --no-build-isolation -e .
-    pytest --cov --cov-fail-under=0 --cov-append --cov-config pyproject.toml pgtrigger/
+    pytest --cov --cov-fail-under=0 --cov-append --cov-config pyproject.toml {posargs:pgtrigger/}
     # There are some tests that must run independently of the original test suite because of making
     # dynamic models
-    pytest --cov --cov-fail-under=0 --cov-append --cov-config pyproject.toml -m independent pgtrigger/
+    pytest --cov --cov-fail-under=0 --cov-append --cov-config pyproject.toml -m independent {posargs:pgtrigger/}
```

Comment:
this allows passing arguments through to pytest, e.g. for running tests with pytest's pdb integration I used:

```
DATABASE_URL=postgres://super:dbpassword@localhost poetry run tox -e py311-django42 -- -- --pdb
```

Two `--` are required to tell poetry, and then tox, to pass the args through.